survey/survey-result-data-offering_shacl.ttl

```diff
@@ -0,0 +1,46 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix survey: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/survey/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+survey:SurveyResultDataOfferingShape
+	a sh:NodeShape ; 
+	sh:targetClass survey:SurveyResultDataOffering ; 
+	sh:property [
+		sh:path survey:surveyStartTime ; 
+		sh:datatype xsd:dateTime ;  
+		skos:example "'Date 1'" ; 
+		sh:minCount 1 ; 
+		sh:maxCount 1 ;
+		sh:message "Validation of survey start time failed." ;
+		sh:name "Survey start time" ;
+		sh:description "When the survey was started." ;
+	] ;
+	sh:property [
+		sh:path survey:surveyCloseTime ; 
+		sh:datatype xsd:dateTime ;  
+		skos:example "'Date 1'" ; 
+		sh:minCount 1 ; 
+		sh:maxCount 1 ;
+		sh:message "Validation of survey close time failed." ;
+		sh:name "Survey close time" ;
+		sh:description "When the survey was closed." ;
+	] ;
+	sh:property [
+		sh:path survey:belongsTo ; 
+		# sh:class survey:SurveyServiceOffering ;
```

Comment:
@robertschubert I had to comment out this line for the checks to pass, let me know if there is a better solution

Reply:
also now the pipeline passes correctly but there two errors which I dont know where the problem is:

Check syntax of all changed Turtle filesUnable to process file command 'env' successfully.
--
Check syntax of all changed Turtle filesInvalid format 'survey/survey-service-offering_shacl.ttl'

Reply:
@phi981 I guess the reason is because the prefix of the ontology is still not correct:

<img width="451" alt="image" src="https://github.com/GAIA-X4PLC-AAD/ontology-management-base/assets/70939848/29f10190-db07-4c5a-9ab0-b154824b1904">

Reply:
I changed the ontology and the shacl and extended the pipeline to read both instance files at once because there is a connection from ResultOffering to ServiceOffering.
Pipelines are running now.